extensions/dicom-pdf/src/id.js

```diff
@@ -0,0 +1,7 @@
+const id = 'org.ohif.dicom-pdf';
+
+export default id;
+
+const SOPClassHandlerName = 'dicom-pdf';
+const SOPClassHandlerId = `${id}.sopClassHandlerModule.${SOPClassHandlerName}`;
```

Comment:
we don't need to insert extension scope in modules, ExtensionManager will handle it internally.

Reply:
Leaving as is because the full ID's are used for referencing things sometimes.